README.md

```diff
@@ -37,21 +34,93 @@ $ ./spin --help
 
 After you follow the [quickstart document](https://developer.fermyon.com/spin/quickstart/),
 you can follow the
-[Rust](https://developer.fermyon.com/spin/rust-components/) or [Go](https://developer.fermyon.com/spin/go-components/)
+[Rust](https://developer.fermyon.com/spin/rust-components/), [JavaScript](https://developer.fermyon.com/spin/javascript-components), [Python](https://developer.fermyon.com/spin/python-components), or [Go](https://developer.fermyon.com/spin/go-components/)
 language guides, and the [guide on configuring Spin applications](https://developer.fermyon.com/spin/configuration/).
 
-After you build your application, run it using Spin:
+Below is an example of using the `spin` CLI to create a new Spin Python application, then adding a JavaScript component:
+
+```bash
+# Create a new Spin application based on the Python language template.
+$ spin new http-py hello-python
+# Add a new JavaScript component based on the language template.
+$ spin add http-js goodbye-javascript
+```
+
+Running the `spin add` command will generate the proper configuration for our component and add it to the [`spin.toml` manifest file](https://developer.fermyon.com/spin/manifest-reference). For example, here is the `spin.toml` section for our Python component:
```

Comment:
The manifest reference is a pretty heavyweight place to drop a reader into.  What do you think of linking to the "Writing Applications" page, which covers a lot of manifest authoring but is hopefully more readable?